### PR TITLE
Added Confirmation when Channel is Set

### LINKED ISSUE #2952: [mesheryctl] Add confirmation when channel is set
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
When running `meheryerctl channel set`, there is no user confirmation indicating the channel set.  

#### Desired Behavior
A confirmation message along the lines of `Set channel <edge/stable/edge-version/stable-version>` should be shown.

#### Screenshots/Logs
Current state:
image

#### Implementation
Use `log.Info` in  `setCmd` in`internal/cli/root/system/channel.go` to display the message.

---

#### Contributor Resources

- [mesheryctl Contributing Guide](https://github.com/layer5io/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by DARK-art108**
**Description**

It is an initial Draft PR for review and fixes.

This PR fixes #2952 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/system/channel.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ var setCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-
+		log.Info("Channel Set to %s", channelNameSeperated)
```
Comment (navendu-pottekkat) [inline mesheryctl/internal/cli/root/system/channel.go:133]:
```suggestion
		log.Infof("Channel set to %s-%s", channelNameSeperated[0], channelNameSeperated[1])
```

**Review by navendu-pottekkat — CHANGES_REQUESTED** [review]
@DARK-art108 Please check out the suggestion and check it out locally.

**Comment by leecalcote** [comment]
@DARK-art108 is this still a draft PR or is it ready for review?

**Comment by DARK-art108** [comment]
@leecalcote This is a draft PR, I have opened it to get review.

**Comment by leecalcote** [comment]
@DARK-art108, ok, in that case, you'll want to remove the draft state by clicking "Ready for Review". A draft state PR means that you're working on it and are not ready for people to look at it. When you've marked it Ready for Review (or just submitted it without draft flagged), then maintainers will take a look.

**Comment by navendu-pottekkat** [comment]
@DARK-art108 @leecalcote I have left a review above, please check it out

**Review by leecalcote — APPROVED** [review]
Thank you, @DARK-art108 and @navendu-pottekkat 👍

**Pull request merged**